src/source-control/GitSourceControl.ts

```diff
@@ -295,7 +295,10 @@ export class GitSourceControl implements SourceControl {
 
       // Update refs in our hashMap
       this.repo = produce(this.repo, (draftRepo) => {
-        const draftTargetCommit = draftRepo.commits.get(targetCommitHash)!;
+        const draftTargetCommit = nullthrows(
+          draftRepo.commits.get(targetCommitHash)!,
```

Comment:
Nit: we don't need to do the TS force unwrap anymore

```suggestion
          draftRepo.commits.get(targetCommitHash),
```